samples/README.md

```diff
@@ -67,18 +74,30 @@ To run the word count sample in Dataproc cluster, follow the steps:
    gcloud config set dataproc/region $REGION
    ```
 <!-- TODO: set up bots to update jar version -->
```

Comment:
Is there an issue for this TODO?  If so, why isn't it mentioned, if not, why not?

Reply:
This TODO is actually outdated, the jar version is passed by CONNECTOR_VERSION as environment vars. Deleted.